services/ui-src/src/components/cards/EntityCard/EntityCardBottomText.tsx

```diff
@@ -0,0 +1,96 @@
+// components
+import { Box, Flex, Text } from "@chakra-ui/react";
+// utils
+import { AnyObject, ModalDrawerEntityTypes } from "types";
+
+export const EntityCardBottomText = ({
+  entityType,
+  entityCompleted,
+  formattedEntityData,
+}: Props) => {
+  switch (entityType) {
+    case ModalDrawerEntityTypes.ACCESS_MEASURES:
+      return (
+        <>
+          {entityCompleted ? (
```

Comment:
Thought: There's an argument to be made that this component should be dumb like `EntityCardTopText`. But that would require us to keep the unfinished message outside of it. But since every card has that and we could just map it through the json, that might be the way to go here.

Reply:
Good call - moved out